ultrasonic/src/main/java/org/moire/ultrasonic/view/UpdateView.java

```diff
@@ -14,6 +14,12 @@
 import java.util.Collection;
 import java.util.WeakHashMap;
 
+/**
+ * A View that is periodically refreshed
+ * @deprecated
+ * Use LiveData to ensure that the content is up-to-date
+ **/
+@Deprecated
```

Comment:
This is debatable, but I'm not a big fan of using Deprecated for tracking our own work to be done. There is the Issue list for that, and also we can use TODOs for smaller things. Deprecated should be used by API or library developers to warn their users of planned changes.
In our own code it will just add unnecessary warnings, which may bug future developers for years if they aren't fixed (because sometimes they require lots of work).